frontend/public/components/persistent-volume.jsx

```diff
@@ -8,7 +8,12 @@ import { Kebab, LabelList, navFactory, ResourceKebab, SectionHeading, ResourceLi
 const { common } = Kebab.factory;
 const menuActions = [...common];
 
-const PVStatus = ({pv}) => <StatusIconAndText status={pv.status.phase} />;
+export const pvPhase = pv => pv.status.phase;
```

Comment:
```suggestion
export const pvPhase = (pv: K8sResourceKind): string => pv.status.phase;
```

Reply:
the file is stil `jsx` so we cannot add types

Reply:
> the file is stil `jsx` so we cannot add types

Ah right, please ignore :)

Reply:
> the file is stil `jsx` so we cannot add types

Nit: technically, you could with `/** @type {declaration_goes_here} */` comments :smiley: